### DISCUSSION TITLE
dwarfs is not present in the pre-built release dwarfs-0.9.8-Linux-x86_64-clang.tar.xz

### DISCUSSION THREAD

**Discussion by DoKoB0512** [mhx/dwarfs#212] [Q&A] [answered]
The pre-built release dwarfs-0.9.8-Linux-x86_64-clang.tar.xz does not contain the dwarfs inside the bin folder. I don't know where to get that or how can I install that on my own.

![image](https://github.com/mhx/dwarfs/assets/123281216/f9496f22-5e92-4b20-afd3-64f64ebdf7ed)

**Comment by silentnoodlemaster** [comment, marked as answer]
it's in sbin

**Reply by DoKoB0512** [reply]
Thank you!
